go/test/endtoend/recovery/unshardedrecovery/recovery.go

```diff
@@ -185,100 +185,100 @@ func TestMainImpl(m *testing.M) {
 }
 
 // TestRecoveryImpl does following
-// - create a shard with primary and replica1 only
-// - run InitShardPrimary
-// - insert some data
-// - take a backup
-// - insert more data on the primary
-// - take another backup
-// - create a recovery keyspace after first backup
-// - bring up tablet_replica2 in the new keyspace
-// - check that new tablet does not have data created after backup1
-// - create second recovery keyspace after second backup
-// - bring up tablet_replica3 in second keyspace
-// - check that new tablet has data created after backup1 but not data created after backup2
-// - check that vtgate queries work correctly
+// 1. create a shard with primary and replica1 only
+//   - run InitShardPrimary
+//   - insert some data
+//
+// 2. take a backup
+// 3.create a recovery keyspace after first backup
+//   - bring up tablet_replica2 in the new keyspace
+//   - check that new tablet has data from backup1
+//
+// 4. insert more data on the primary
+// 5. take another backup
+// 6. create a recovery keyspace after second backup
+//   - bring up tablet_replica3 in the new keyspace
+//   - check that new tablet has data from backup2
```

Comment:
I don't see where we take steps 5 and 6. Am I missing it? I only see two backups taken, and those seem to line up with steps 2,3 and 8,9.

Reply:
updated